react-charts/line-chart/step6.md

```diff
@@ -1,56 +1,55 @@
-Now we'll add axis labels to the line chart...
+Add a legend to the line chart.
 
-## Task
+1) <strong>Make sure the App.js file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>

```

Comment:
Update all of this to follow what is in area-chart:

<img width="304" alt="Screen Shot 2019-12-05 at 10 44 18 PM" src="https://user-images.githubusercontent.com/20118816/70294229-cd0bb180-17b0-11ea-87a0-96af5a331c72.png">